examples/twitter.amp.html

```diff
@@ -20,30 +20,45 @@
       layout="responsive"
       data-tweetid="638793490521001985">
       <blockquote placeholder class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I only needed to change some CSS. <a href="http://t.co/LvjLbjgY9F">pic.twitter.com/LvjLbjgY9F</a></p>&mdash; Malte Ubl (@cramforce) <a href="https://twitter.com/cramforce/status/638793490521001985">September 1, 2015</a></blockquote>
+      <div fallback>
+        Something happened..
```

Comment:
Agree with this. I tested with `amp-img` and it _tries_ to do the same flow as described here but is somewhat broken: created https://github.com/ampproject/amphtml/issues/10321